### ISSUE TITLE
Add some more maintainer(s)

### ISSUE THREAD

**Issue by etanot** [xalanq/cf-tool#100]
Relevant comments https://github.com/xalanq/cf-tool/issues/93#issuecomment-629919580 and https://github.com/xalanq/cf-tool/issues/93#issuecomment-629958362

If you think project maintainer should add some more maintainers, please upvote this issue.
@xalanq, if you have some free time, could you please consider adding some more maintainers to this project? Thank you. :)

**Comment by etanot** [comment]
Relevant comment https://github.com/xalanq/cf-tool/issues/93#issuecomment-629958362

> create an organisation (since multiple tools are to be made under the same name).

Cool... We could await few days for @xalanq's reply and then create an organization. Or @infixint943, you can create an organization right away, since you have plans to expand this project for multiple OJs; and later you could discuss about with @xalanq; which ever solution you like.

**Comment by infalmo** [comment]
That sounds ideal. But I have 1 major issue at hand.....
What should I name the organisation? XDXD
Cause I'm not able to think of a solid name for the project. Any suggestions?

**Comment by etanot** [comment]
How about **CP Tools**, or **CP CLI** etc? Or something similar which contains **CP** term in it.

**Comment by infalmo** [comment]
Hmmm. Actually, we can change organisation name later.
But cp is already taken by most other tools. Let me think till today evening for a good name
Also, are you interested in contributing to the project? I'd really appreciate it!

**Comment by etanot** [comment]
@infixint943 Yes, I'm; but I don't know Golang. I have to start it from scratch, which is not possible right now, since right now I'm doing CP; but I can help you with other things like with documentation, reporting issue etc. And thanks for opportunity. :)

**Comment by infalmo** [comment]
I've created the organization https://github.com/cp-tools. I'd be glad if you could help in the documentation of the tool

**Comment by etanot** [comment]
@infixint943 Just ping me, whenever you need some help.
